src/config.rs

```diff
@@ -118,21 +124,21 @@ impl SafeFillConfig for SafeConfig {
         // will be done to modify base.
         let mut base = default_config();
 
-        if self.powersave_under.is_some() {
-            base.powersave_under = self.powersave_under.unwrap();
+        if let Some(pu) = self.powersave_under {
+            base.powersave_under = pu;
         }
```

Comment:
Can you explain to me how this is safer? Who does wrapping it in a Some work??

Reply:
It is a syntax that defines `pu` only if `self.powersave_under` is Some as the unwrapped value of powersave_under. This is similar to the previous code but is easier to understand and without the unwrap written out by the user.